mobilenetv2-burn/src/model/imagenet.rs

```diff
@@ -0,0 +1,1034 @@
+use burn::tensor::{backend::Backend, Device, Tensor};
```

Comment:
It is used in resnet and now in mobilenet. I too feel that we could classify the models say into image classification module and then have common utils, then have the models themselves be sub modules in it. It would also be helpful in future.

Reply:
Yeah it's used in both ResNet and MobileNet.. but right now the models are released separately. We should probably finish the discussion in #16 for loading weights since this is where the common code lies.

Not sure what the best approach would be.